src/package-request.js

```diff
@@ -391,6 +412,7 @@ export default class PackageRequest {
           wanted,
           latest,
           url,
+          isOldHash,
```

Comment:
Flow gets mad at this because, although it looks like a plain JS object is being made here, it actually wants an object that fits the `Dependency` type, which is defined in `src/types.js`. That type definition does not have a field named `isOldHash`, so to fix it, we need to add it:

_in src/types.js_ add `isOldHash` as an optional boolean field:

```
export type Dependency = {
  // ...other existing properties omitted...
  isOldHash?: boolean,
};
```